refinery/core/tests.py

```diff
@@ -667,12 +673,16 @@ def test_verify_dataset_deletion_if_analysis_run_upon_it(self):
                           DataSet.objects.get,
                           name="dataset_with_analysis")
 
-    def test_isa_archive_deletion(self):
+    @mock.patch('core.models.DataSet.get_investigation_links')
```

Comment:
If we have to mock this many times it may be good to think about adding a reference in a common `setUp()` and just `.start()` the mock in these three tests.

Something like:
```
def setUp(self):
    self.get_investigation_links_mock = mock.patch('core.models.DataSet.get_investigation_links', return_value=None)

def testA(self):
    self.get_investigation_links_mock.start()
```